modules/core/exported/callbacks.go

```diff
@@ -0,0 +1,19 @@
+package exported
+
+type CallbackPacketDataI interface {
```

Comment:
```suggestion
type CallbackPacketData interface {
```
I think `ibcexported.CallbackPacketData` should make it apparent it is an interface?